homework/src/index.js

```diff
@@ -30,15 +34,96 @@
     return elem;
   }
 
+  function addRow(label, value, tbody) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { class: 'label', text: label });
+    createAndAppend('td', tr, { text: value });
+  }
+
+
+  function renderRepository(info, container) {
+    container.innerHTML = '';
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, {
+      text: 'Repository: ',
+    });
+    const repository = createAndAppend('td', tr1);
+    createAndAppend('a', repository, {
+      href: info.html_url,
+      target: '_blank',
+      text: info.name
+    });
+
+    const descriptions = `${info.description} `;
+    addRow('Description: ', descriptions, tbody);
+    const forks = `${info.forks} `;
+    addRow('Forks: ', forks, tbody);
+    const updated = `${info.updated_at} `;
+    addRow('Updated: ', updated, tbody);
+
+  }
+
+  function renderContributor(info, container) {
```

Comment:
Since this function deals with rendering the contributors (note the plural), a more accurate name would be `renderContributors` (note the plural). A better name for the parameter `info` (a rather meaningless name) would be `contributors`.